plugins/ros-pac-man/src/ucrmInfo.php

```diff
@@ -0,0 +1,101 @@
+<?php
+    
+    // Specify the UCRM config file for the plugin
+    
+    $file = __DIR__ . "/../data/config.json";
+    
+    // Specify UCRM info file
+    
+    $uFile = __DIR__ . "/../ucrm.json";
+    
+    // Get the UCRM config file options
+    
+    $uConfigInfo = file_get_contents($uFile);
+    
+    // Decode the UCRM file from JSON to an array
+    
+    $uConfigs = json_decode($uConfigInfo,true);
+    
+    $ucrmPublicURL = $uConfigs['ucrmPublicUrl'];
+    
+    // Get the config file options
+    
+    $configInfo = file_get_contents($file);
+    
+    // Decode the file from JSON to an array
+    
+    $configs = json_decode($configInfo,true);
+    
+    $pmIP = $configs['pmIP'];
+    
+    $pmAdminUser = $configs['pmAdminUser'];
+    
+    $pmAdminPass = $configs['pmAdminPass'];
+    
+    $pm2IP = $configs['pm2IP'];
+    
+    $pm2AdminUser = $configs['pm2AdminUser'];
+    
+    $pm2AdminPass = $configs['pm2AdminPass'];
+    
+    // Specific UCRM URL for private use
+    
+    $ucrmURL = $ucrmPublicURL."api/v1.0";
+    
+    // UCRM API key with read or write capability
+    
+    $ucrmKey = "YOUR_KEY_HERE";
```

Comment:
you can get this (`pluginAppKey`) from the automatically created `ucrm.json` file in the root of plugin, take a look at docs for more info https://github.com/Ubiquiti-App/UCRM-plugins/blob/master/docs/file-structure.md#ucrmjson

this plugin is unusable as of now without editing the code

Reply:
Edited to pull the key from ucrm.json.